doc/nrf/known_issues.rst

```diff
@@ -679,6 +679,14 @@ Closing sockets
 Multiprotocol Service Layer (MPSL)
 ==================================
 
+.. rst-class:: v1-4-99-dev1 v1-4-2 v1-4-1
+
+DRGN-15223: Do not use `CONFIG_SYSTEM_CLOCK_NO_WAIT` with nrf53.
+
+.. rst-class:: v1-4-2 v1-4-1
+
+DRGN-15176: `CONFIG_SYSTEM_CLOCK_NO_WAIT` will have no effect when the Low Frequency Clock is started before initializing MPSL.
```

Comment:
You need more information here - see the other entries.
The first line is the heading/summary, and after that you should describe the issue and, if possible, a workaround.

Reply:
Done

Reply:
Hmm, I was actually thinking about both entries. ;)

For this one, how about:
```
DRGN-15176: `CONFIG_SYSTEM_CLOCK_NO_WAIT` is ignored when Low Frequency Clock is started before initializing MPSL

  If the application starts the Low Frequency Clock before calling :c:func:`mpsl_init()`, the clock configuration option :option:`CONFIG_SYSTEM_CLOCK_NO_WAIT` has no effect.
  MPSL will wait for the Low Frequency Clock to start.
  
  **Workaround:** When :option:`CONFIG_SYSTEM_CLOCK_NO_WAIT` is set, make sure to initialize MPSL before starting the Low Frequency Clock.
```

Reply:
I need to rephrase the workaround sentence. It makes no sense for the application to start LF clock after MPSL is initialized. It is even forbidden in this case.

How about: " **Workaround:** When :option:`CONFIG_SYSTEM_CLOCK_NO_WAIT` is set, do not start the Low Frequency Clock before MPSL initialization."?

I am not sure what I can explain for DRGN-15223, we just do not guarantee that it works. The workaround is to not use it. Could you suggest rephrasing with this info in mind?

Reply:
So that means you should not start the LF clock at all - correct?
Then just use: 
```
**Workaround:** When :option:`CONFIG_SYSTEM_CLOCK_NO_WAIT` is set, do not start the Low Frequency Clock.
```

Reply:
Updated.
Assuming I do not need to change anything for the first entry about nrf53